docs/_includes/css/forms.html

```diff
@@ -908,30 +908,42 @@ <h2 id="forms-control-sizes">Control sizing</h2>
   <div class="bs-example bs-example-control-sizing" data-example-id="form-control-height-sizes">
     <form>
       <div class="controls">
-        <input class="form-control input-lg" type="text" placeholder=".input-lg">
-        <input type="text" class="form-control" placeholder="Default input">
-        <input class="form-control input-sm" type="text" placeholder=".input-sm">
+        <label class="sr-only" for="height-size-1">Height Size 1</label>
+        <input class="form-control input-lg" id="height-size-1" type="text" placeholder=".input-lg">
+        <label class="sr-only" for="height-size-2">Height Size 2</label>
+        <input class="form-control" id="height-size-2" type="text" placeholder="Default input">
+        <label class="sr-only" for="height-size-3">Height Size 3</label>
+        <input class="form-control input-sm" id="height-size-3" type="text" placeholder=".input-sm">
 
```

Comment:
the placeholder attribute is exposed by browsers through the accessibility API as a name for the control, in the absence of other naming methods (``<label>``, ``title``, ``aria-label``, ``aria-labelledby`` etc). This is also valid, and supported by AT (provided the placeholder text makes sense as a label, which in this case it does - though, by the nature of these controls as demos to show actual styling, it may not be the prettiest)